README.md

```diff
@@ -1,3 +1,42 @@
-# lxdexporter_golang
+# lxd_exporter
 
-Set Environment variable LXD_SOCKET to use different path ( default is /var/lib/lxd/unix.socket )
+[![Build Status](https://travis-ci.org/nieltg/lxd_exporter.svg?branch=master)](https://travis-ci.org/nieltg/lxd_exporter)
```

Comment:
Please change this url

Reply:
I had changed these URLs. Please take a look.